### PR TITLE
Skip PV encryption tests if Vault config is not present

### PR DESCRIPTION
**Pull request by rachael-george**
Fixes: #5048 

Signed-off-by: rachael-george <rgeorge@redhat.com>

### CODE REVIEW

**Comment by rachael-george** [comment]
Verified PR locally:
```
$ cat ocs-ci/data/auth.yaml |grep vault
$ 

$ run-ci --cluster-name rgeorge-d22 --cluster-path /home/rgeorge/OCS4-scripts/clusters/aws/  --ocs-version 4.9 tests/manage/pv_services/test_rbd_pv_encryption.py --dev-mode

tests/manage/pv_services/test_rbd_pv_encryption.py::TestRbdPvEncryption::test_rbd_pv_encryption[v1] SKIPPED [ 50%]
tests/manage/pv_services/test_rbd_pv_encryption.py::TestRbdPvEncryption::test_rbd_pv_encryption[v2] SKIPPED [100%]

========================================== 2 skipped in 0.01s ==========================================
----------------------------------------- live log collection ------------------------------------------
13:40:58 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc get clusterversion -n openshift-storage -o yaml
13:41:02 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc get clusterversion -n openshift-storage -o yaml
tests/manage/pv_services/pvc_clone/test_encrypted_rbd_pvc_clone.py::TestEncryptedRbdClone::test_pvc_to_pvc_clone[v1] SKIPPED [ 50%]
tests/manage/pv_services/pvc_clone/test_encrypted_rbd_pvc_clone.py::TestEncryptedRbdClone::test_pvc_to_pvc_clone[v2] SKIPPED [100%]

========================================== 2 skipped in 5.86s ==========================================
----------------------------------------- live log collection ------------------------------------------
13:42:03 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc get clusterversion -n openshift-storage -o yaml
13:42:07 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc get clusterversion -n openshift-storage -o yaml
SKIPPED [ 50%]
tests/manage/pv_services/pvc_snapshot/test_encrypted_rbd_pvc_snapshot.py::TestEncryptedRbdBlockPvcSnapshot::test_encrypted_rbd_block_pvc_snapshot[v2] SKIPPED [100%]

========================================= 2 skipped in 19.95s ==========================================
```

File: ocs_ci/framework/pytest_customization/marks.py [hunk 1]
```diff
@@ -205,6 +205,10 @@
     reason="Test runs ONLY on OSD or ROSA cluster",
 )
 
+kms_config_required = pytest.mark.skipif(
+    config.ENV_DATA.get("vault") is not True,
```
Comment (jilju) [inline ocs_ci/framework/pytest_customization/marks.py:209]: Is ```vault``` already defined?

**Review by jilju — DISMISSED** [review]

File: ocs_ci/framework/pytest_customization/marks.py (hunk 1 above)
Comment (rachael-george) [reply ocs_ci/framework/pytest_customization/marks.py]: It was not defined under `ENV_DATA`. Changed the skip condition accordingly.

**Review by ebenahar — APPROVED** [review]

**Review by sidhant-agrawal — APPROVED** [review]
lgtm

**Pull request merged**